madmom/ml/nn/layers.py

```diff
@@ -344,6 +344,144 @@ def activate(self, data):
         return out
 
 
+class GRUCell(object):
+    """
+    Cell as used by GRU layers proposed in [1].
+
+    Parameters
+    ----------
+    weights_in_to_hidden_update : numpy array, shape ()
+        Weights.
+    weights_in_to_hidden_update : numpy array, shape ()
+        Weights.
+    bias : scalar or numpy array, shape ()
+        Bias.
+    activation_fn : numpy ufunc, optional
+        Activation function.
+
+    References
+    ----------
+    .. [1] Cho, Kyunghyun, et al,
+           "On the properties of neural machine translation: Encoder-decoder
+           approaches",
+           arXiv preprint arXiv:1409.1259 (2014).
```

Comment:
Ok. However, in most madmom classes, the authors come first: 
E.g.,
    .. [1] Sebastian Böck and Markus Schedl,
           "Enhanced Beat Tracking with Context-Aware Neural Networks",
           Proceedings of the 14th International Conference on Digital Audio
           Effects (DAFx), 2011.
    .. [1] Emilia Gómez,
           "Tonal Description of Music Audio Signals",
           PhD thesis, Universitat Pompeu Fabra, Barcelona, Spain, 2006.
    .. [1] Filip Korzeniowski and Gerhard Widmer,
           "Feature Learning for Chord Recognition: The Deep Chroma Extractor",
           Proceedings of the 17th International Society for Music Information
           Retrieval Conference (ISMIR), 2016.